packages/address-consts/CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-<!-- ## Unreleased -->
+## Unreleased
+
+### Breaking Change
+
+- Drop support for node 12. Remove wildcard export. Update `@shopify/browserslist-config`. [[#2277](https://github.com/Shopify/quilt/pull/2277)]
```

Comment:
Consumers don't care about the change to `@shopify/browserslist-config`, they care about the impact that it has, What do you think to using this as the changelog line?


```suggestion
- Drop support for node 12 and Safari 10, 11 and 12. Remove wildcard export in exports field. [[#2277](https://github.com/Shopify/quilt/pull/2277)]
```